src/container/TreeContainer/Base/TreeNode.ts

```diff
@@ -1,7 +1,12 @@
+export const enum TreeNodeColor {
+  // eslint-disable-next-line no-unused-vars
```

Comment:
Maybe you can remove this comments?

Reply:
I think it's a bug in eslint. There's nothing we can do about this

Reply:
Changed to scope disable. Please make sure it looks better
```typescript
export const enum TreeNodeColor {
  /* eslint-disable no-unused-vars */
  RED = 1,
  BLACK = 0
  /* eslint-enable no-unused-vars */
}
```